src/test/fuzz/p2p_transport_serialization.cpp

```diff
@@ -17,16 +19,25 @@
 #include <optional>
 #include <vector>
 
+namespace {
+
+std::vector<std::string> g_all_messages;
+
 void initialize_p2p_transport_serialization()
 {
     SelectParams(ChainType::REGTEST);
+    g_all_messages = getAllNetMessageTypes();
+    std::sort(g_all_messages.begin(), g_all_messages.end());
```

Comment:
Why this sort here? Anybody remembers? It is only used like:

```cpp
g_all_messages[v % g_all_messages.size()]
```

where `v` is a random number. The sort seems unnecessary?

The question arised in: https://github.com/bitcoin/bitcoin/pull/29421#discussion_r1490653235